test/command/test-base.rb

```diff
@@ -122,6 +122,81 @@ def test_pretty_print
     end
   end
 
+  class CovnertToElasticsearchFormatTest < self
+    sub_test_case("non_target_command") do
+      def test_select_command
+        select = Groonga::Command::Base.new("select",
+                                            :table => "Users",
+                                            :filter => "age<=30",
+                                            :output_type => "json")
+        assert_equal(nil,
+                     select.to_elasticsearch_format)
+      end
+    end
+
+    sub_test_case("single_record") do
+      def test_brackets_format
+        load = Groonga::Command::Base.new("load",
+                                          :table => "Site",
+                                          :values => <<-VALUES)
+  [
+  ["_key","title"],
+  ["http://example.org/","This is test record 1!"]
+  ]
+        VALUES
+        assert_equal("{\"index\":{\"_index\":\"groonga\",\"_type\":\"Site\"}}\n" +
+                     "{\"_key\":\"http://example.org/\",\"title\":\"This is test record 1!\"}",
```

Comment:
It seems that reducing escape is better to maintain testcase.

How about like this?
```
        expected = <<OUTPUT
{"index":{"_index":"groonga","_type":"Site"}}
{"_key":"http://example.org/","title":"This is test record 1!"}
OUTPUT
        assert_equal(expected.chomp, load.to_elasticsearch_format)
```